controlplane/internal/dal/dal.go

```diff
@@ -428,6 +428,34 @@ func (d *DAL) loadDeployment(ctx context.Context, deployment sql.GetLatestDeploy
 	return out, nil
 }
 
+type Metric struct {
+	RunnerKey    ulid.ULID
+	StartTime    uint64
```

Comment:
Is it a good idea to have these as `time.Time`? If not, we should add the unit to the name (eg. `StartTimeNs` or `StartTimeMs`).

Reply:
By that I mean that I generally prefer to have values as strongly typed as possible where possible, and convert them to untyped forms at the last possible moment.